src/components/CategoryFilter.js

```diff
@@ -1,12 +1,28 @@
 import "./CategoryFilter.css";
 
-export default function CategoryFilter() {
+export default function CategoryFilter({ onCategoryFilterChange }) {
+  function handleCategoryFilterClick(event) {
+    const categoryFilterValue = event.target.value;
+    onCategoryFilterChange(categoryFilterValue);
+  }
+
   return (
     <div className="CategoryFilter">
-      <select name="category" id="category" className="category-search-bar">
-        <option disabled="disabled" selected="selected" className="label">
+      <select
+        name="category"
+        id="category"
+        className="category-search-bar"
+        onClick={handleCategoryFilterClick}
+      >
+        <option
+          disabled="disabled"
+          selected="selected"
```

Comment:
aren't these booleans?

Reply:
true true. let me get rid of the unnecessary code